applications/ConstitutiveLawsApplication/custom_constitutive/small_strain_isotropic_damage_3d.cpp

```diff
@@ -124,59 +129,85 @@ void SmallStrainIsotropicDamage3D::InitializeMaterial(
 //************************************************************************************
 //************************************************************************************
 
-void SmallStrainIsotropicDamage3D::InitializeMaterialResponseCauchy(ConstitutiveLaw::Parameters& rValues)
+void SmallStrainIsotropicDamage3D::FinalizeMaterialResponsePK2(
+    ConstitutiveLaw::Parameters& rParametersValues)
 {
+    Vector internal_variables(1);
+    this->CalculateStressResponse(rParametersValues, internal_variables);
+    mStrainVariable = internal_variables[0];
 }
 
-//************************************************************************************
-//************************************************************************************
+void SmallStrainIsotropicDamage3D::FinalizeMaterialResponseCauchy(
+    ConstitutiveLaw::Parameters& rParametersValues)
+{
+    FinalizeMaterialResponsePK2(rParametersValues);
```

Comment:
this does not look correct to me. 

i would call the correct one in here... (i mean, ending with PK2, Cauchy...)